GetStream/stream-chat-swift
# [CIS-1580] Fix "Only visible to you" shown when deleted messages visible for all users

### 🔗 Issue Link
CIS-1580

### 🎯 Goal
Fix the "Only visible to you" label being displayed when the `ChatClientConfig` is set to `.alwaysVisible`.

### 🛠 Implementation

**Applied Solution:**
The current implementation is done in the `ChatMessageListVC`. When the layout options are provided, we override them and remove the `.onlyVisibleToYou` label in case the config is not the `.onlyVisibleToCurrentUser`. 

**Alternative Solution:**
Probably it would make more sense to implement this directly in the `ChatMessageLayoutOptionsResolver`, but this would add a bit of complexity because we would need to inject the `ChatClientConfig` in the `ChatMessageLayoutOptionsResolver`. This could be a bit error-prone since we can't inject it in the constructor, otherwise, we would break the API, and it would be weird to have an optional chat client config in the resolver. For these reasons, the first implementation was the most simple one considering our options.

### 🧪 Testing

1. Launch the Demo App
2. Tap on the Configuration button and set the `.alwaysVisible` for deleted messages.
3. Enter a channel and delete a message
4. Deleted messages should not show the "Only Seen by You” label

### ☑️ Contributor Checklist
